styles/main.css

```diff
@@ -108,44 +93,41 @@ ul {
 
 .park-page {
   display: none;
+  margin: 2rem;
+  grid-template-columns: 1fr 1fr;
+  gap: 20px;
 }
 
 .park-page.active {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-template-rows: 10% 90%;
-  min-height: 100vh;
-  background: black;
+  display: block;
 }
 
 .park-name {
-  background: #2d4b1e;
-  color: white;
+  color: black;
   grid-area: 1 / 1 / 1 / 3;
 }
 
 .left {
   grid-area: 2 / 1 / 3 / 2;
-  background: #2d4b1e;
 }
 
 .right {
   grid-area: 2 / 2 / 3 / 3;
-  background: #2d4b1e;
 }
 
 #back {
   display: block;
+  margin-top: 20px;
+  margin-left: 2rem;
 }
 
 #back.active {
   display: none;
 }
 
 #next {
-  background-color: violet;
   margin-top: 20px;
-  margin-right: 20px;
+  margin-right: 2rem;
```

Comment:
What is rem? When should I use rem as opposed to px? Sorry if you covered this because it sounds somewhat familiar!

Reply:
Its a unit that is relative to the root element's font-size. I just sort of get into a habit of replacing pixels with rem nowadays. They are useful because you can change the root's font-size, and then all of the other values in your project will also scale. Great for accessibility, where a user my scale the font-size by default.
This is a pretty good video discussing best practices with units: https://www.youtube.com/watch?v=N5wpD9Ov_To
I changed this to change the overall value, not just swap units.

Reply:
Ok sounds good. Thank you for the video and explanation !